managed/services/telemetry/config.default.yml

```diff
@@ -704,7 +704,7 @@ telemetry:
 
   - id: AdvisorsChecksDisabled
     source: PMMDB_SELECT
-    query: settings->'sass'->'disabled_stt_checks' as disabled_checks from settings;
+    query: settings->'sass'->'disabled_advisors' as disabled_checks from settings;
```

Comment:
it looks like a duplicate of metric from line 37, isn't it?

Reply:
Yeah it is, but after we enabled advisors by default, it made sense to invert it so we can track users that disabled advisors.